RubeesEat/Controllers/BillController.cs

```diff
@@ -45,13 +45,15 @@ public Results<BadRequest<string>, RedirectHttpResult> Create()
                 break;
             }
 
-            Person? person;
             if (!Guid.TryParse((id), out var personGuid))
             {
                 return TypedResults.BadRequest("Person Guid not valid");
             }
+            
+            Person? person = personRepository.GetById(personGuid);
+
 
-            if (!personRepository.TryGetById(personGuid, out person))
+            if (person == null)
```

Comment:
Please remove the empty lines between the variable declaration and the `if`. They are related to each other and should stand side by side.